dts/arm/nuvoton/npcx.dtsi

```diff
@@ -52,6 +52,14 @@
 		status = "okay";
 	};
 
+	/** Dummy node of IOs that have leakage current. The user can override
```

Comment:
nit: does doxygen parse these? If not I think this should open with the normal single-star thing (`/*`)

Reply:
Done.